skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/Features.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.pensjon.brev.skribenten
+
+import kotlin.reflect.KProperty
+
+object Features {
```

Comment:
Kanskje vi bør tenke på å få inn unleash etterhvert.

Reply:
Jeg ville ikke gjøre det nå fordi vi vet egentlig ikke om det er noe spesielt behov for det.